queue_services/business-pay/devops/vaults.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "vault": "shared",
+        "application": [
+            "nats"
+        ]
+    },
+    {
+        "vault": "entity",
+        "application": [
+            "postgres-legal",
+            "entity-pay",
+            "nats-pay",
+            "sentry"
+        ]
+    }
```

Comment:
{
        "vault": "gcp-queue",
        "application": [
            "a083gt-dev",
            "topics"... etc.
        ]
    }

Reply:
need the new section in here for business.. which has the audience endpoints as well